### PR TITLE
Update CDC tests to print out the stderr when tests fail

### PR DESCRIPTION
**Pull request by NKTomHaygarth**
When running the tests for the Core Dump Composer the error output was not being printed. This was making things a little difficult to track down where the tests might be failing if the Composer was hitting an error path.

This PR just adds `println!("{}", String::from_utf8_lossy(&cdc.stderr));` underneath all locations where `cdc.stdout` is already getting printed in the tests for the Composer

### CODE REVIEW

**Comment by No9** [comment]
Thank you very much for this PR and happy to land it if we can resolve the clippy issue.

Please can you either rebase this to this PR https://github.com/IBM/core-dump-handler/pull/116 
or wait until 116 landed an rebase to main. 

Thank again for the contribution.

**Comment by NKTomHaygarth** [comment]
You're welcome. I'm away until Monday if the PR #116 isn't merged in by then I can rebase to that when I'm back

**Review by No9 — APPROVED** [review]
Simple change so LGTM - @NKTomHaygarth Can you rebase and we will pull this in for the next release

**Comment by No9** [comment]
Hey @NKTomHaygarth Just doing some house keeping.
do you still want this in or can I close this?

**Comment by NKTomHaygarth** [comment]
Hey @No9, sorry things got a little busy towards the end of last year so I had to shelve some of the stuff I was doing. I will look into rebasing this PR later this week and hopefully get it ready to merge in.

**Comment by No9** [comment]
No worries - thanks for coming back to it.

**Review by No9 — APPROVED** [review]

**Comment by No9** [comment]
Thanks for following up on this @NKTomHaygarth

**Pull request merged**
